src/index.html

```diff
@@ -4,65 +4,375 @@
+            </div>
             </div>
+
+            <br><br><br>
+            <p>Validator Selection</p>
+
+
+
+
+            <button id="boton" class="pool">
+            <p class="pool-name">Just The Kus</p>
+            
+            </button>
+
+            <div class="tooltip" id="tooltip">
+            If you are not already staking with this account, we recommend the full set to
+            ensure that you get the top rewards.
+            </div>
+
+
+            <button class="pool">
+            <p class="pool-name">The Kus Partner Set</p>
+            
+            </button>
+
+      </div>
+    </div>
+  </div>
+
+  <div id="myModalD" class="modal">
+
+    <!-- Modal content -->
+    <div class="modal-content">
+      <div class="modal-header">
+        <span class="close">&times;</span>
+        <h2>Vote</h2>
+        <ul>
+          <li>
+            <input type="radio" id="f-option" name="selector">
+            <label for="f-option">Vote</label>
+            <div class="check"></div>
+          </li>
+
+          <li>
+            <input type="radio" id="s-option" name="selector">
+            <label for="s-option">UnVote</label>
+            <div class="check">
+              <div class="inside"></div>
+            </div>
+          </li>
+        </ul>
+
+      </div>
+      <div class="modal-body">
+        <p style="margin-top: 80px;">Partners with:</p>
+        <img id="a" style="max-width: 110px;" src="logos/lucky.png" alt="">
+        <div class="columns is-vcentered">
+            <!-- <div class="column">
+                <div class="field">
+                    <input id="delegationSwitch" type="checkbox" name="switchLarge" class="switch is-large is-rtl">
+                    <label for="delegationSwitch">I want to delegate my votes to you!</label>
+                </div>
+            </div> -->
+            <div class="column">
+                <!-- Callout explaining the rules we're following when voting on your behalf -->
+                <!-- <div class="notification is-warning is-light">
+                    Delegate votes to the Kusamarian voting collective
+            </div> -->
         </div>
-    </section>
-    <div class="is-divider"></div>
-    <section>
+    </div>
+        <div class="input-group">
+          <input value="" class="form-control" type="text" name="text-1542372332072" id="text-1542372332072"
+            required="required" placeholder="DOT Amount">
+          <label for="text-1542372332072">DOT Amount</label>
+        </div>
+
+        <div class="filters">
+          <select id="main_select_services" class="select-club-services" name="club-services">
+            <option value=".abrikosovaya" selected>7 Days <br> x1</option>
+            <option value=".bratskaya">14 Days <br> x2</option>
+            <option value=".lesi-ukrainki">28 Days<br> x3</option>
+            <option value=".lesi-ukrainki">56 Days <br> x4</option>
```

Comment:
Is this a copy-paste bug?
Options names seem to be repeated from here onwards.

Also: as much as I appreciate the choice the naming choice here, I can imagine the code to be better maintainable (esp. by people not knowing the language) if the option names would be semantically-meaningful.